Update webroutingsettings (#5317)
umbraco/UmbracoDocs: 10/umbraco-cms/reference/configuration/webroutingsettings.md

## Try skip IIS custom errors

> Defines the value of Response.TrySkipIisCustomErrors when an error (404, 400, 500...) is encountered. You probably want it to be true in order to prevent IIS from displaying its own 404 or 500 pages. Instead have your own page displayed.

Reviewer: Not sure about this final sentence here - how about we do something like this instead: 

```suggestion
Defines the value of Response.TrySkipIisCustomErrors when an error (404, 400, 500...) is encountered. In order to prevent IIS from displaying its own 404 or 500 pages, set this to `true` to have your own page displayed.
```

Author: Seems good to me! 😄
